src/oscar/templates/oscar/dashboard/layout.html

```diff
@@ -121,6 +121,14 @@ <h1>{% block headertext %}{% endblock %}</h1>
 
 {% endblock %}
 
+{% block cdn_scripts %}
+  {{ block.super }}
+  {% if use_less and debug %}
+    {# Load the on-the-fly less compiler. Never do this in production. #}
+    <script src="//cdnjs.cloudflare.com/ajax/libs/less.js/2.5.3/less.min.js"></script>
```

Comment:
The latest version on the CDN is 3.9.0 - is there a reason you haven't used that?

Reply:
Well I have just used the version from ``src/oscar/templates/oscar/layout.html``. I thought that might be more consistent.

Reply:
Could you please update both to use the latest version?